descriptors/ModuleDescriptor-template.json

```diff
@@ -283,6 +283,47 @@
         }
       ]
     },
+    {
+      "id": "scheduled-notice-storage",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/scheduled-notice-storage/scheduled-notices",
+          "permissionsRequired": ["scheduled-notice-storage.scheduled-notices.collection.get"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/scheduled-notice-storage/scheduled-notices/{id}",
+          "permissionsRequired": ["scheduled-notice-storage.scheduled-notices.item.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/scheduled-notice-storage/scheduled-notices",
+          "permissionsRequired": ["scheduled-notice-storage.scheduled-notices.item.post"]
+        },
+        {
+          "methods": ["POST"],
```

Comment:
Is this endpoint for batch updating of records?

Reply:
Yes

Reply:
This does not appear to be used by the [implementation of CIRC-322](https://github.com/folio-org/mod-circulation/pull/258/files), why has it been implemented in this first piece of work?

Reply:
Initially it was decided that CIRC-322 needs bulk save endpoint to save multiple scheduled notices in one request. But during implementation it was decided to use usual POST to save entities separately. Such decision was made because in most cases amount of notices not as big to require bulk save (about 1-3). So I think I should remove this endpoint for now as it is not used.

Reply:
That's great, thank you. It gives us time to better establish the pattern for batch record creation or updating.